conda_build/noarch_python.py

```diff
@@ -55,8 +65,11 @@ def handle_file(f, d):
         os.unlink(path)
 
     # The presence of .so indicated this is not a noarch package
-    elif f.endswith('.so'):
-        _error_exit("Error: Shared object file found: %s" % f)
+    elif f.endswith(('.so', '.dll', '.pyd', '.exe', '.dylib')):
+        if f.endswith('.exe') and f.startswith('Scripts'):
```

Comment:
I think this is still not quite right.  People should install executables into Library/bin, but not everyone knows that.  If someone installs a non-entry-point exe in Scripts, we should complain.

how about something like:

    if not os.path.exists(os.path.splitext(f)[0] + "-script.py"):
        # barf because this is an exe in the wrong place